### PR TITLE
Atmos logging

### PR DESCRIPTION
**Pull request by gbasood**
Because why the hell are these things NOT logged

### CODE REVIEW

File: code/ATMOSPHERICS/pipe/construction.dm [hunk 1]
```diff
@@ -502,11 +500,11 @@ var/list/heat_pipes = list(PIPE_HE_STRAIGHT, PIPE_HE_BENT, PIPE_JUNCTION)
 			P =new /obj/machinery/atmospherics/pipe/layer_manifold(src.loc)
 
 		if(PIPE_LAYER_ADAPTER)
-			testing("src.loc = [src.loc]")
```
Comment (gbasood) [inline code/ATMOSPHERICS/pipe/construction.dm:505]: This line seems like old ass debug output that we don't need so I removed it

File: code/ATMOSPHERICS/pipe/construction.dm [hunk 2]
```diff
@@ -436,7 +435,6 @@ var/list/heat_pipes = list(PIPE_HE_STRAIGHT, PIPE_HE_BENT, PIPE_JUNCTION)
 
 		if(PIPE_JUNCTION)
 			P=new /obj/machinery/atmospherics/pipe/simple/heat_exchanging/junction( src.loc )
-			investigation_log(I_ATMOS,"was created by [user]/([user.ckey]) at [formatJumpTo(loc)].")
```
Comment (gbasood) [inline code/ATMOSPHERICS/pipe/construction.dm:439]: And these lines got moved to further down
This way all pipe constructions are logged, and they're also only logged if they actually succeed in building

**Comment by ghost** [comment]
The answer is always "because no-one could bother"

Logging code has barely any visible impact on the game, although it's very useful for admins if someone did something really fucking complex and far-fetched to griff the whole server

**Comment by gbasood** [comment]
The reason this PR exists is because some dumbass did something not even that far fetched
Just sayin

**Comment by ghost** [comment]
With that being said, Virology wasn't logged for the longest time, which meant that once a virus started killing everyone the admin's best guess was to Crew Manifest and find the Viro

I was always wondering why every single person on the server got away with playing auto-antag Virologist for so long

**Pull request merged**